backend/app/Models/Account.php

```diff
@@ -20,6 +20,7 @@ class Account extends Model
         'user_id',
         'account_type',
         'account_name',
+        'account_number'
```

Comment:
my code changes for this file

- adding the trait to the model
`use Uuid, UniqueCodeTrait;`

fn getUniqueCodeKeyName to specify to the trait which key to use
```
    /**
     * Indicates if the model's ID is auto-incrementing.
     *
     * @var bool
     */
    public $incrementing = false;

    /**
     * function for UniqueCodeTrait to retrieve the column 
     * name for to generate the unique code for
     */
    public function getUniqueCodeKeyName()
    {
        return "account_number";
    }

    public function user()
    {
        return $this->belongsTo(User::class);
    }
```